apps/prairielearn/src/pages/instructorIssues/instructorIssues.sql

```diff
@@ -70,42 +70,26 @@ WITH
       )
       AND (
         $filter_query_text::text IS NULL
-        OR to_tsvector(
-          concat_ws(' ', q.directory, u.uid, i.student_message)
-        ) @@ plainto_tsquery($filter_query_text::text)
+        OR to_tsvector(concat_ws(' ', q.qid, u.uid, i.student_message)) @@ plainto_tsquery($filter_query_text::text)
       )
   )
 SELECT
-  i.id AS issue_id,
-  format_date_iso8601 (
-    now(),
-    coalesce(ci.display_timezone, c.display_timezone)
-  ) AS now_date,
-  format_date_iso8601 (
-    i.date,
-    coalesce(ci.display_timezone, c.display_timezone)
-  ) AS formatted_date,
+  i.* AS issue_id,
+  now() AS now,
   ci.short_name AS course_instance_short_name,
   ci.id AS course_instance_id,
+  COALESCE(ci.display_timezone, c.display_timezone) AS display_timezone,
```

Comment:
The timezone is retrieved here instead of from `res.locals` because the page may be listing issues from different course instances with different timezones.